app/controllers/landing_pages_controller.rb

```diff
@@ -0,0 +1,148 @@
+class LandingPagesController < ApplicationController
+  # Needs to be aligned with values in set_org_types
+  GROUP_TYPES = %w[multi_academy_trust local_authority].freeze
+
+  skip_before_action :authenticate_user!
+  before_action :set_marketing_case_studies
+  before_action :set_org_types, only: [:book_demo, :more_information]
+
+  def index
+    redirect_to find_out_more_campaigns_path
+  end
+
+  def mat_pack
+    redirect_to url_for(controller: :resource_files, action: :download, serve: :inline, id: 30)
+  end
+
+  def school_pack
+    redirect_to url_for(controller: :resource_files, action: :download, serve: :inline, id: 29)
+  end
+
+  def demo_video
+    redirect_to 'https://www.youtube.com/watch?v=x2EeYWwdEpE'
```

Comment:
do you want to use the no cookies domain?

Reply:
thats for embeds. video doesn't display properly (no controls, description, etc) if you try and watch via no-cookies version.